dspace/dspace.dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM janx/ubuntu-dev
+
+# Get dependencies (JDK 8, Maven, Ant, Tomcat, PostgreSQL)
+RUN sudo apt-get update -q \
+ && sudo apt-get upgrade -qy \
+ && sudo apt-get install -qy \
+  ant \
+  maven \
+  postgresql \
+  tomcat8
+
+# Get source code
+RUN git clone https://github.com/dspace/dspace /home/user/dspace
+WORKDIR /home/user/dspace
+
+# Make db maybe do this after
+#RUN createuser --username=postgres --no-superuser --pwprompt dspace \
+# && createdb --username=postgres --owner=dspace --encoding=UNICODE dspace \
+# && psql --username=postgres dspace -c "CREATE EXTENSION pgcrypto;"
+
+# Setup config
+COPY local.cfg /home/user/dspace/dspace/config
+RUN sudo chown user:user /home/user/dspace/dspace/config/local.cfg
+
+RUN mvn package \
+ && cd dspace/target/dspace-installer \
+ && ant fresh_install
+
+# Symlink some files
+RUN ln -s /home/user/dspace/webapps /home/user/tomcat8/webapps
```

Comment:
I still don't quite get when the `/home/user/tomcat8` folder gets created. Is it during `mvn package` or `ant fresh_install`?

Reply:
I believe `/home/user/tomcat8` gets created when `tomcat8` is installed.
As for `/home/user/dspace/webapps` that gets created after `ant`